purebred.cabal

```diff
@@ -73,15 +73,13 @@ library
                      , text
                      , process
                      , mime-mail
-                     , network
                      , directory
                      , bytestring
                      , time
                      , case-insensitive
                      , optparse-applicative >= 0.13
                      , xdg-basedir
                      , filepath
-                     , unix
```

Comment:
Did you use anything to find these?

Reply:
There exists https://hackage.haskell.org/package/packunused but on this occasion I just
tried removing each of the deps that I wasn't sure were used, and if it still built, well...  :)

Reply:
Ah right... cheers.